cli/aws_orbit/remote_files/delete.py

```diff
@@ -25,16 +27,16 @@
 _logger: logging.Logger = logging.getLogger(__name__)
 
 
-def delete_image(args: Tuple[str, ...]) -> None:
-    _logger.debug("args %s", args)
-    context: "Context" = ContextSerDe.load_context_from_ssm(env_name=args[0], type=Context)
+def delete_image(env_name: str, image_name: str) -> None:
+    _logger.debug("env_name: %s, image_name: %s", env_name, image_name)
+    context: "Context" = ContextSerDe.load_context_from_ssm(env_name=env_name, type=Context)
     _logger.debug("context.name %s", context.name)
-    if len(args) == 2:
-        image_name: str = args[1]
-    else:
-        raise ValueError("Unexpected number of values in args.")
-
-    env.deploy(context=context, eks_system_masters_roles_changes=None)
-    _logger.debug("Env changes deployed")
-    ecr.delete_repo(repo=f"orbit-{context.name}-{image_name}")
-    _logger.debug("Docker Image Destroyed from ECR")
+
+    @remotectl.remote_function("orbit", codebuild_role=context.toolkit.admin_role)
+    def delete_image(env_name: str, image_name: str) -> None:
+        env.deploy(context=context, eks_system_masters_roles_changes=None)
+        _logger.debug("Env changes deployed")
+        ecr.delete_repo(repo=f"orbit-{context.name}-{image_name}")
```

Comment:
The ECR repo name generation is not correct....f"orbit-{context.name}/{image_name}